File: Daily Challanges/2026-07-July/2026-07-27.py
import math


def is_pronic(n):
    x = math.isqrt(n)
    return x * (x + 1) == n


print(is_pronic(6))
print(is_pronic(15))
print(is_pronic(12))
print(is_pronic(132))
print(is_pronic(80))
print(is_pronic(0))

File: Daily Challanges/2026-07-July/2026-07-27.js
function isPronic(n) {
  const x = Math.floor(Math.sqrt(n));
  return x * (x + 1) === n;
}

console.log(isPronic(6));
console.log(isPronic(15));
console.log(isPronic(12));
console.log(isPronic(132));
console.log(isPronic(80));
console.log(isPronic(0));
